lib/requests.js

```diff
@@ -149,6 +158,12 @@ QueryRequest.prototype.write = function (encoder) {
   //v3: <query>
   //      <consistency><flags>[<n>[name_1]<value_1>...[name_n]<value_n>][<result_page_size>][<paging_state>][<serial_consistency>][<timestamp>]
   var frameWriter = new FrameWriter(types.opcodes.query);
+  var headerFlags = this.options.traceQuery ? types.frameFlags.tracing : 0;
```

Comment:
Another possible thing I noticed is that you currently can't get at the prepared statement custom payload if you wanted to act on it in some way in the client.  I'm not sure if there is a use case for wanting to look at that data, but thought I'd point that out.